### PR TITLE
Extend OrganizationStylingService

### PR DESCRIPTION
**Pull request by scavnickyj**
JIRA: TNT-923

---

Supported PR commands:

| Command                  | Description             |
| ------------------------ | ----------------------- |
| `ok to test`             | Re-run standard checks  |
| `extended test`          | BackstopJS tests        |
| `extended check sonar`   | SonarQube tests         |
| `extended check cypress` | Cypress E2E tests       |
| `extended check plugins` | Dashboard plugins tests |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `check-extended-cypress` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

File: libs/sdk-backend-spi/src/organization/styling/index.ts [hunk 1]
```diff
@@ -10,5 +10,9 @@ import { IThemeMetadataObject, ObjRef } from "@gooddata/sdk-model";
 export interface IOrganizationStylingService {
     getThemes(): Promise<IThemeMetadataObject[]>;
```
Comment (kandl) [inline libs/sdk-backend-spi/src/organization/styling/index.ts:11]: We should probably add some docs for these methods.

File: libs/sdk-backend-tiger/src/backend/organization/styling.ts [hunk 2]
```diff
@@ -18,31 +19,48 @@ export class OrganizationStylingService implements IOrganizationStylingService {
...
                     },
-                }),
-            );
-        }
+                },
+            }),
+        );
+    }
+
+    public async deleteActiveTheme(): Promise<void> {
```
Comment (kandl) [inline libs/sdk-backend-tiger/src/backend/organization/styling.ts:59]: This name is probably misleading - it does not delete the theme at all, it just resets some organization setting.
Should it be rather `resetActiveTheme`, `clearActiveTheme`,  `unsetActiveTheme`  or something similar?

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/8967/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/10811/ : SUCCESS in 18m 02s

**Review by kandl — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR2794

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/2563/ : SUCCESS in 1m 38s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/2862/ : SUCCESS in 20s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/1693/ : SUCCESS in 15m 53s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/10814/ : SUCCESS in 20m 40s

**Pull request merged**